src/contractor/contractor.cpp

```diff
@@ -95,10 +98,11 @@ EdgeWeight getNewWeight(IterType new_speed_iter, const double segment_length, Ed
     {
```

Comment:
The intended purpose of this flag is more for visibility into speed updates over lua provided defaults, not necessarily changes of live traffic.